tests/ssl-opt.sh

```diff
@@ -2420,7 +2583,7 @@ run_test    "TLS 1.3, TLS1-3-AES-128-CCM-SHA256, ECDHE-ECDSA, mismatched sig_alg
 # - the expected parameters are selected
 requires_config_enabled MBEDTLS_SSL_PROTO_TLS1_2
 requires_ciphersuite_enabled TLS-ECDHE-RSA-WITH-CHACHA20-POLY1305-SHA256
-requires_config_enabled MBEDTLS_SHA512_C # "signature_algorithm ext: 6"
+requires_hash_alg SHA_512 # "signature_algorithm ext: 6"
```

Comment:
Using tls13-prototype commit of the prototype, this test is run in the full configuration. That's not the case with the changes of this PR. Please have a look. There are some other cases in that case.

Reply:
reverted. But I did not understand the reason.